nexus-tweaks.user.js

```diff
@@ -1144,6 +1144,264 @@ promiseList.push((async () => {
+            let found = false
+            for (let firstLevel in Badges) {
+                for (let secondLevel in Badges[firstLevel]) {
+                    if (Badges[firstLevel][secondLevel].includes(bT)) {
+                        if (!(firstLevel in classifiedBadges)) classifiedBadges[firstLevel] = {}
+                        if (!(secondLevel in classifiedBadges[firstLevel])) classifiedBadges[firstLevel][secondLevel] = []
+                        classifiedBadges[firstLevel][secondLevel].push(bT)
+                        found = true
+                        continue
+                    }
+                }
+                if (found) continue
+            }
+            if (!found) {
+                if (!classifiedBadges.Others) classifiedBadges.Others = []
+                classifiedBadges.Others.push(bT)
+            }
+        }
+
+        return classifiedBadges
+    }
+
+    function printClassifiedBadges(classifiedBadges, badgesNode) {
+        function L1(text, parent) {
+            const categoryNode = parent.appendChild(document.createElement('div'))
+            categoryNode.style.backgroundColor = "#dddddd"
+            const titleNode = categoryNode.appendChild(document.createElement('div'))
+            const contentNode = categoryNode.appendChild(document.createElement('div'))
+            titleNode.style.fontWeight = "900"
+            titleNode.textContent = text
+            titleNode.title = "Click to expand/collapse"
+            titleNode.onclick = () => { if (contentNode.style.display != "none") contentNode.style.display = "none"; else contentNode.style.display = "initial" }
+            return contentNode
+        }
+        function L2(text, parent) {
+            const newNode = document.createElement('div')
+            newNode.style.backgroundColor = "#eeeeee"
+            newNode.style.fontWeight = "normal"
+            newNode.textContent = text
+            return parent.appendChild(newNode)
+        }
+        function L3(text, parent) {
+            const newNode = document.createElement('div')
+            newNode.style.backgroundColor = "#ffffff"
+            newNode.style.fontWeight = "normal"
+            newNode.textContent = text
+            return parent.appendChild(newNode)
+        }
+        function orderObject(unordered, ordering) {
+            return Object.keys(unordered).sort(
+                (a,b) => (Object.keys(ordering).indexOf(a) < Object.keys(ordering).indexOf(b) ? -1 : 1)
+            ).reduce((obj, key) => { obj[key] = unordered[key]; return obj }, {})
+        }
+
+        if (classifiedBadges.Career) {
+            const nodeL1 = L1("Career:", badgesNode)
+            Object.entries(orderObject(classifiedBadges.Career, Badges.Career)).forEach(e => {
+                const badge6 = ["Damage Dealt", "Damage Taken", "HP Healed"]
+                const nodeL2 = L2("\u2003" + e[0] + " [" + e[1].length + (badge6.includes(e[0]) ? "/6]" : "/5]"), nodeL1)
```

Comment:
Please add in-line comments for each unicode codepoint to improve readability:
https://google.github.io/styleguide/jsguide.html#non-ascii-characters